validators_test.md

```diff
@@ -12,32 +12,33 @@ Every validator test is represented by a dedicated type (struct) which implement
 
 ```go
 type Validator interface {
-	Name() string
-	Run(utils.MetaBundle) (bool, error)
-	SucceedingCandidates() []utils.MetaBundle
-	FailingCandidates() []utils.MetaBundle
+    Name() string
+    Run(MetaBundle) (bool, error)
```

Comment:
```suggestion
    Run(types.MetaBundle) (bool, error)
```

Reply:
good catch, fixed to -> `Run(types.MetaBundle) types.ValidatorResult`

removed the commented out code and pasted in the ticket as a comment instead -> https://issues.redhat.com/browse/MTSRE-390